Model/Cart.php

```diff
@@ -231,9 +246,9 @@ public function preparePayload($storeId, $extraData = [])
             // Required key/value pairs not in the payload by default.
             $payload['API_KEY']= $this->apiConfig->getApiKey($store->getWebsiteId());
             $payload['distinct_user_hash'] = $payload['CURRENT_PIXLEE_USER_ID'];
-            $payload['ecommerce_platform'] = 'magento_2';
-            $payload['ecommerce_platform_version'] = '2.0.0';
-            $payload['version_hash'] = $this->getVersionHash();
+            $payload['ecommerce_platform'] = Pixlee::PLATFORM;
+            $payload['ecommerce_platform_version'] = $this->productMetadata->getVersion();
```

Comment:
how does thiss version get set?

Reply:
This is the Magento method used by vendor/magento/module-backend/Block/Page/Footer.php:56 getMagentoVersion() to get the Magento version that is displayed in the admin footer.